charts/app-config/values-integration.yaml

```diff
@@ -792,9 +792,12 @@ govukApplications:
       enabled: false
     extraEnv:
       - name: PRIMARY_UPSTREAM
-        value: "http://draft-content-store/"
+        value: "http://draft-content-store-mongo-main/"
       - name: SECONDARY_UPSTREAM
         value: "http://draft-content-store-postgresql-branch/"
+    service:
+      selector:
+        app: draft-content-store
```

Comment:
optional: usually [nesting is discouraged](https://helm.sh/docs/chart_best_practices/values/#flat-or-nested-values) (i.e. prefer flat names unless there's some overriding lesser-of-evils kind of reason) so I would name this just `serviceSelector` (only the "app" deployment has a service) and probably make it the whole selector object rather than special-casing it to be the value of the `app` label (since that's easier to understand as well as being more flexible).

Shout if you'd like an example for that (I'm a little short on time today — sorry).

Reply:
I've updated it to use a flat value (I think this was what you meant?)